src/frequenz/client/dispatch/__main__.py

```diff
@@ -37,22 +36,23 @@
 DEFAULT_DISPATCH_API_PORT = 50051
 
 
-def get_client(host: str, port: int) -> Client:
+def get_client(host: str, port: int, key: str) -> Client:
```

Comment:
I suggest making these keywork-only for extra clarity now that we are taking 2 `str`, otherwise one could accidentally do a `get_client(key, port, host)` and maybe when that connection fails the `host` get logged, potentially leaking the key.

```suggestion
def get_client(*, host: str, port: int, key: str) -> Client:
```

Reply:
For a local only singular-used once called function? Seems unnecessary to me..
Only to have a call like 

`get_client(host=host, port=port, key=key)` that really is not more informative than before ;)

Reply:
OK, it is private I agree.

Reply:
Well, yes. It was clear before. It was not really used anywhere except in this one instance, in that instance the case was very very clear, so I don't really see any advantage for this occasion but only costs to add something.. with no value (here)